File: bubbleSort.cpp
#include<iostream>
using namespace std;
void printArray(int arr[], int size){
    for(int i=0; i<size; i++){
        cout<<arr[i]<<" ";
    }
}
void bubble_sort(int arr[], int size){
    for(int i=0; i<size-1; i++){
        for(int j=0; j<size-i-1; j++){
           if(arr[j]>arr[j+1]){
             swap(arr[j], arr[j+1]);
            }
        }
    }
}
int main(){
    int arr[] = {5, 9, 2, 7, 1};
    int size = sizeof(arr)/sizeof(arr[0]);
    cout<<"Unsorted array is: ";
    printArray(arr, size);
    bubble_sort(arr, size);
    cout<<"\nSorted array is: ";
    printArray(arr, size);
    return 0;
}

File: BubbleSort.py
def bubble_sort(arr):
  for i in range(len(arr)-1):
    for j in range(0, len(arr)-i-1):
      if arr[j] > arr[j+1]:
        swapped = True
        arr[j], arr[j+1] = arr[j+1], arr[j]
  return arr
print(bubble_sort([10, 7, 8, 9, 1, 5]))